src/typescript/frontend/src/app/global.css

```diff
@@ -11,7 +11,7 @@
   }
 
   .Toastify__progress-bar {
-    height: 20px;
+    height: 7px;
```

Comment:
I couldn't find the figma for it for the life of me. But regardless it's too big, I think it needs to be smaller since it was encroaching upon the text. We can show Kirsten and run it by her

Reply:
Per eng call, will be addressed in ECO-2250